src/app/containers/RadioSchedule/index.test.jsx

```diff
@@ -12,10 +15,30 @@ const localRadioScheduleEndpoint = service => {
   return `${localServiceURL}/bbc_${service}_radio/radioschedule.json`;
 };
 
-const renderRadioScheduleContainer = service =>
+const getToggleState = enabled => ({
+  local: { radioSchedule: { enabled } },
+  test: { radioSchedule: { enabled } },
+});
+
+const renderRadioScheduleContainer = (service, radioScheduleToggle = false) =>
   act(async () => {
     ReactDOM.render(
-      <RadioScheduleContainer endpoint={localRadioScheduleEndpoint(service)} />,
+      <ToggleContext.Provider
```

Comment:
This looks pretty similar to this https://github.com/bbc/simorgh/pull/5275/files#diff-1a29ba9d2679ef0c669dda294e9a396bR26-R37, perhaps we can create a helper function and just pass in a `toggle-state `and `endpoint` to reduce the duplication.

Reply:
Yes, they are similar - but I think combining them will create an inconsistency as in Simorgh tests and stories do not share components. A component is created for tests, and another for stories